README.md

```diff
@@ -191,3 +191,21 @@ pnpm run dev
 
 The (minimal) web application will be accessible at: http://localhost:5173
 The api workbench will be accessible at: http://localhost:5174
+
+## 4 Forking
+
+The following commands can be used to rename "protoapp" to "myproject" in a fork
+of this repo:
+
+```
+git mv adl/{protoapp,myproject}
+git mv rust/adl/src/gen/{protoapp,myproject}
+git mv rust/server/src/bin/{protoapp,myproject}-server.rs
+git mv rust/server/src/bin/{protoapp,myproject}-tools.rs
+git mv ts/adl/src/{protoapp,myproject}
+sed -i -e 's|protoapp|myproject|g'  $(git ls-files)
+sed -i -e 's|Protoapp|Myproject|g'  $(git ls-files)
+sed -i -e 's|ProtoApp|MyProject|g'  $(git ls-files)
```

Comment:
There's really only one instance of 'ProtoApp' perhaps it should just be changed to 'Protoapp' and then simplify this with 

```suggestion
sed -i -e 's|Protoapp|MyProject|g'  $(git ls-files)
```